py-polars/src/lazy/dsl.rs

```diff
@@ -1422,6 +1422,27 @@ impl PyExpr {
             .into()
     }
 
+    fn date_round(&self, every: &str, offset: &str) -> Self {
```

Comment:
Can we move this to the datatime namespace in polars lazy?

The goal is to have no anonymous functions anymore.

Reply:
Ok will get to this in the next couple of days. 

Not 100% sure what you mean (v new to rust) just to check: polars/polars-lazy/polars-plan/src/dsl/dt.rs is the datetime namespace?